.github/workflows/snapshot.yml

```diff
@@ -0,0 +1,32 @@
+name: "Snapshot"
+on:
+  workflow_dispatch:
+    inputs:
+      snapshotVersion:
+        description: "Snapshot version"
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin' # This is the Ubuntu Default
+          java-version: '17'
+          
+      - name: Build library
+        run: 'bash ./gradlew clean :library:assembleRelease'
+          
+      - name: Release library
+        env:
+          MOBILE_MAVENCENTRAL_USER: ${{ secrets.MOBILE_MAVENCENTRAL_USER }}
+          MOBILE_MAVENCENTRAL_PASSWORD: ${{ secrets.MOBILE_MAVENCENTRAL_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
+          ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEYID }}
+        run: "bash ./gradlew publishReleasePublicationToSonatypeRepository -DSNAPSHOT_VERSION=${{ github.event.inputs.snapshotVersion }} publishNoopPublicationToSonatypeRepository -DSNAPSHOT_VERSION=${{ github.event.inputs.snapshotVersion }}
+        --max-workers 1 closeAndReleaseStagingRepositories"
```

Comment:
we were already supporting [versioning for snapshots](https://github.com/Telefonica/android-logger/blob/main/build.gradle#L40) but it wasn't being exposed through any Github workflow: